client/src/components/MultiJurisdictionAudit/Progress/JurisdictionDetail.tsx

```diff
@@ -35,6 +35,12 @@ const FileStatusTag = ({
   }
 }
 
+const prettyCvrFileType = (cvrFileType: CvrFileType) =>
+  ({
+    [CvrFileType.DOMINION]: 'Dominion',
```

Comment:
I'm confused by the square brackets around the `CvrFileType.DOMINION`... You can't key on the raw CVR file type?

Reply:
square brackets are the notation for using a dynamic value as a property name. I can't remember why I did it this way but it looks like TS can also figure it out if i just use the CVR file type strings directly, so i'll change it